diff --git tensorflow/tensorflow.bzl tensorflow/tensorflow.bzl
index 9fa8c9d7628..dfe58b309ad 100644
--- tensorflow/tensorflow.bzl
+++ tensorflow/tensorflow.bzl
@@ -1473,7 +1473,7 @@ def tf_gen_op_wrapper_py(
     # Make a py_library out of the generated python file.
     if not generated_target_name:
         generated_target_name = name
-    py_deps = [clean_dep("//tensorflow/python/framework:for_generated_wrappers_v2")]
+    py_deps = []
     if extra_py_deps:
         py_deps += extra_py_deps
     py_lib_rule(
